src/Documents.jl

```diff
@@ -239,6 +245,7 @@ function Document(plugins = nothing;
         root     :: AbstractString   = Utilities.currentdir(),
         source   :: AbstractString   = "src",
         build    :: AbstractString   = "build",
+        working_dir::Union{Symbol, AbstractString}  = :build,
```

Comment:
Even more bikeshed -- how about `workdir`? Just that all the other keywords are pretty short and this would make it more consistent with them I feel.

Reply:
I'm fine with this!